src/middlewares/logRequest.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response, NextFunction } from 'express';
+
+// Logs some basic info for the specified request to the console.
+export default function logRequest(request: Request, _: Response, next: NextFunction): void {
```

Comment:
TIL: How to specify an unused parameter in TypeScript. Used to have to rely on casting to void, `(void)response` in my old C/C++ days to silence compiler warnings.